src/Monolog/Handler/MongoDBHandler.php

```diff
@@ -11,41 +11,66 @@
 
 namespace Monolog\Handler;
 
+use MongoDB\Driver\BulkWrite;
+use MongoDB\Driver\Manager;
+use MongoDB\Client;
 use Monolog\Logger;
 use Monolog\Formatter\NormalizerFormatter;
 
 /**
  * Logs to a MongoDB database.
  *
- * usage example:
+ * Usage example:
  *
- *   $log = new Logger('application');
- *   $mongodb = new MongoDBHandler(new \Mongo("mongodb://localhost:27017"), "logs", "prod");
+ *   $log = new \Monolog\Logger('application');
+ *   $client = new \MongoDB\Client('mongodb://localhost:27017');
+ *   $mongodb = new \Monolog\Handler\MongoDBHandler($client, 'logs', 'prod');
  *   $log->pushHandler($mongodb);
  *
- * @author Thomas Tourlourat <thomas@tourlourat.com>
```

Comment:
This class was essentially rewritten from the 1.x version, so I didn't see much reason to keep `@author` around.